src/components/sandbox/views/signed-out-view.tsx

```diff
@@ -33,7 +35,15 @@ const ItemCaption = (props: BoxProps) => (
 
 export const SignedOutView: React.FC<any> = () => {
   const { doOpenAuth } = useConnect();
+  const [installed, setInstallStatus] = useState(false);
 
+  const extInstalled = IS_BROWSER && typeof window.BlockstackProvider !== 'undefined';
```

Comment:
FYI I literally just changed it to `StacksProvider` but I still have `BlockstackProvider` for compatibility.